docs/content/en/docs/languages/_index.md

```diff
@@ -43,7 +43,7 @@ The following pre-built environments are currently available for use in Fission:
 | ------------------------------------ | ------------------------- | -------------------------- | --- | --- | --- |
 | NodeJS                               | `fission/node-env`        | `fission/node-builder`     | O   | O   | O   |
 | Python 3                             | `fission/python-env`      | `fission/python-builder`   | O   | O   | O   |
-| Go                                   | see [here]({{% relref "go.md" %}}#add-the-go-environment-to-your-cluster) for more info | | O   | O   | O   |
+| Go                                   | see [here]({{% relref "./go.md" %}}#add-the-go-environment-to-your-cluster) for more info | | O   | O   | O   |
```

Comment:
We will have to make this change for all sublinks?

Reply:
when I add zh folder, some `relref` throw error because of  ambiguity, surely hugo works without changing some of the links, but  I'm not sure which changes are must and which not, so in my opinion, why not use relative path as much as possible to avoid ambiguity?